README.md

```diff
@@ -7,5 +12,8 @@ These instructions assume you know how to use Git, NPM and Bower.
 
 - Clone the repo using `git clone https://github.com/blackbaud/barkbaud-ui.git`
 - Run `npm install`, `bower install`, and `grunt build` from the directory where you cloned the repo
-- Launch the sample site from a web server (the NPM package [http-server](https://www.npmjs.com/package/http-server) makes it easy to launch a web server from any location on your local drive) and navigate to `build/#/artists` to test it out.
+- Launch the sample site from a web server (the NPM package [http-server](https://www.npmjs.com/package/http-server) makes it easy to launch a web server from any location on your local drive; see below).
+    - Type: `npm install http-server -g`.
+    - To run the server on a specific port, type: `http-server -p 5000 -a localhost` (port 8080 should also work).
```

Comment:
Ah, I know why I put that in there. Terminal provides the URL of the server's location, and to avoid any confusion (and to guarantee that the user went to "localhost"), I made it more explicit:

![screen-shot-2016-04-26-at-11 17 16-am](https://cloud.githubusercontent.com/assets/12497062/14823622/304eadf8-0ba1-11e6-95e6-144a0d593229.png)

***This is what the browser looks like if the user "accidentally" goes to http://127.0.0.1:5000/build***:
![screen-shot-2016-04-26-at-11 21 18-am-1](https://cloud.githubusercontent.com/assets/12497062/14823674/5b3a54a4-0ba1-11e6-9146-3b10f222c7d6.png)